packages/wonder-blocks-toolbar/components/toolbar.js

```diff
@@ -0,0 +1,206 @@
+// @flow
```

Comment:
Top-level docstring.

Reply:
Thanks, I added a simple one at the top.